code/game/objects/effects/misc.dm

```diff
@@ -65,6 +65,9 @@
 /obj/effect/abstract/marker/at
 	name = "active turf marker"
 
+/obj/effect/abstract/marker/intercom
```

Comment:
is there not a more fitting file to add to than a misc file

Reply:
this is for consistency with the other abstract marker